Specs/Core/TransformsSpec.js

```diff
@@ -1716,6 +1716,77 @@ describe("Core/Transforms", function () {
     expect(returnedResult).toEqualEpsilon(expected, CesiumMath.EPSILON12);
   });
 
+  it("rotationMatrixFromPositionVelocity works without a result parameter", function () {
+    var matrix = Transforms.rotationMatrixFromPositionVelocity(
+      Cartesian3.UNIT_X,
+      Cartesian3.UNIT_Y
+    );
+    var expected = new Matrix3(0, 0, 1, 1, 0, 0, 0, 1, 0);
+    // using Matrix3.abs to fix the 0/-0 comparisons
```

Comment:
You should be able to replace this with `toEqualEpsilon` throughout.  For example:

```js
expect(matrix).toEqualEpsilon(expected, CesiumMath.EPSILON14);
```

Reply:
Awesome that works. Makes me feel a lot better